README.md

```diff
@@ -43,6 +43,14 @@ Or, for the `mime/lite` version:
     mimelite.getType(...); // (Note `mimelite` here)
     <script>
 
+## Deno
+Despite its name this module can be used not just in node context, but also with [deno](https://deno.land), which is kind of the successor of node.js.
```

Comment:
Nit: Avoid "passive voice".  State things confidently or delete 'em. :-)

Also, the module's name is `mime`.  It's only the repo that's called `node-mime`... and I should probably fix that.

```suggestion
This module can be used with [deno](https://deno.land).
```